src/assets/stylesheets/_navigation.scss

```diff
@@ -303,11 +295,18 @@
     font-size: 22px;
     text-align: center;
     margin: 0 auto;
-    color: $primary;
+    color: $theme-greywhite;
     @include transition(transform 0.2s ease-in-out);
   }
 }
 
+.bg-white {
+  .navbar-search-toggle .fa,
+  .menu-toggle .fa {
+    color: $primary !important;
```

Comment:
* [ ] This sometimes makes the arrow invisible for the selected top menu item, e.g. https://deploy-preview-472--scrivito-example.netlify.app/pages-and-widgets

<img width="554" alt="image" src="https://user-images.githubusercontent.com/986170/165053204-395b218d-5080-40ce-9a78-864dbadbeee1.png">